spec/turn_spec.rb

```diff
@@ -0,0 +1,294 @@
+      turn.pile_cards
+      expect(turn.spoils_of_war).to eq([card1, card2, card5, card4, card3, card6])
+      turn.award_spoils
+      expect(player1.deck.cards.length).to eq(1)
+      expect(player1.deck.cards).to eq([card8])
+      expect(player2.deck.cards.length).to eq(7)
+      expect(player2.deck.cards).to eq([card7, card1, card2, card5, card4, card3, card6])
+    end
+  end
+
+  describe "#mutually assured destruction turn type" do
+    it "will return turn type of mutually_assured_destruction" do
+      card1 = Card.new(:heart, 'Jack', 11)
+      card2 = Card.new(:heart, '10', 10)
+      card3 = Card.new(:heart, '9', 9)
+      card4 = Card.new(:diamond, 'Jack', 11)
+      card5 = Card.new(:heart, '8', 8)
+      card6 = Card.new(:diamond, '8', 8)
+      card7 = Card.new(:heart, '3', 3)
+      card8 = Card.new(:diamond, '2', 2)
+      deck1 = Deck.new([card1, card2, card5, card8])
+      deck2 = Deck.new([card4, card3, card6, card7])
+      player1 = Player.new("Megan", deck1)
+      player2 = Player.new("Aurora", deck2)
+      turn = Turn.new(player1, player2)
+
+      expect(turn.type).to eq(:mutually_assured_destruction)
+
+    end
+
+    it "will return no winner" do
+      card1 = Card.new(:heart, 'Jack', 11)
+      card2 = Card.new(:heart, '10', 10)
+      card3 = Card.new(:heart, '9', 9)
+      card4 = Card.new(:diamond, 'Jack', 11)
+      card5 = Card.new(:heart, '8', 8)
+      card6 = Card.new(:diamond, '8', 8)
+      card7 = Card.new(:heart, '3', 3)
+      card8 = Card.new(:diamond, '2', 2)
+      deck1 = Deck.new([card1, card2, card5, card8])
+      deck2 = Deck.new([card4, card3, card6, card7])
+      player1 = Player.new("Megan", deck1)
+      player2 = Player.new("Aurora", deck2)
+      turn = Turn.new(player1, player2)
+
+      expect(turn.winner).to eq("No Winner")
+
+    end
+
+    it "top 3 cards from each players decks will be removed, will not be added to spoils of war" do
+      card1 = Card.new(:heart, 'Jack', 11)
+      card2 = Card.new(:heart, '10', 10)
+      card3 = Card.new(:heart, '9', 9)
+      card4 = Card.new(:diamond, 'Jack', 11)
+      card5 = Card.new(:heart, '8', 8)
+      card6 = Card.new(:diamond, '8', 8)
+      card7 = Card.new(:heart, '3', 3)
+      card8 = Card.new(:diamond, '2', 2)
+      deck1 = Deck.new([card1, card2, card5, card8])
+      deck2 = Deck.new([card4, card3, card6, card7])
+      player1 = Player.new("Megan", deck1)
+      player2 = Player.new("Aurora", deck2)
+      turn = Turn.new(player1, player2)
+
+      expect(player1.deck.cards).to eq([card1, card2, card5, card8])
+      expect(player2.deck.cards).to eq([card4, card3, card6, card7])
+      expect(turn.winner).to eq("No Winner")
+      turn.pile_cards
+      expect(turn.spoils_of_war).to eq([])
+      expect(player1.deck.cards.length).to eq(1)
+      expect(player1.deck.cards).to eq([card8])
+      expect(player2.deck.cards.length).to eq(1)
+      expect(player2.deck.cards).to eq([card7])
+
+    end
+  end
+end
```

Comment:
Your `Turn` tests are awesome! The only one you missed was testing `award_spoils` for a `:mutually_assured_destruction` scenario. If you had added that test, you would have realized it was a point in your program that had the potential for breaking and would have probably added that condition I suggested earlier.